examples/solids/elasticity.h

```diff
@@ -64,6 +77,7 @@ static const char *const problemTypesForDisp[] = {"Linear elasticity",
                                                   "Hyperelasticity finite strain Initial config Neo-Hookean w/ dXref_dxinit, Grad(u), C_inv, constant storage",
                                                   "Hyperelasticity finite strain Current config Neo-Hookean w/ dXref_dxinit, Grad(u) storage",
                                                   "Hyperelasticity finite strain Current config Neo-Hookean w/ dXref_dxcurr, tau, constant storage",
+                                                  "Hyperelasticity finite strain Initial config Moony-Rivlin w/ dXref_dxinit, Grad(u) storage"
```

Comment:
This is human-readable output. Perhaps we should say `configuration` instead of `config`